src/map/packets/roe_questlog.h

```diff
@@ -0,0 +1,44 @@
+﻿/*
+===========================================================================
+
+  Copyright (c) 2020 - Kreidos | github.com/kreidos
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see http://www.gnu.org/licenses/
+
+===========================================================================
+*/
+
+#ifndef _CROEQUESTLOGPACKET_H
+#define _CROEQUESTLOGPACKET_H
+
+#include "../../common/cbasetypes.h"
+
+#include "basic.h"
+
+/************************************************************************
```

Comment:
If we aren't putting information in these block comments, do we need them?

Reply:
Axe em'!